### PR TITLE
added testcase to detect default method usage

### PR DESCRIPTION
**Pull request by ivadimko**

### CODE REVIEW

File: src/arrayMethodIndexOf.test.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ test('indexOf2 is added to [].__proto__', () => {
     .toBeInstanceOf(Function);
 });
 
+test(`indexOf2 doesn't call default indexOf`, () => {
+  expect([].indexOf2.toString().indexOf('.indexOf'))
```
Comment (mgrinko) [inline src/arrayMethodIndexOf.test.js]: .indexOf(

Comment (mgrinko) [reply src/arrayMethodIndexOf.test.js]: And what will you do with `.indexOf` in `.includes2` implementation?

Comment (ivadimko) [reply src/arrayMethodIndexOf.test.js]: @mgrinko 
image

File: src/arrayMethodIndexOf.js [hunk 2]
```diff
@@ -5,7 +5,7 @@
  *
  */
 function applyCustomIndexOf() {
-  [].__proto__.indexOf2 = function() {
+  [].__proto__.indexOf2 = function(...args) {
```
Comment (roman-mate) [inline src/arrayMethodIndexOf.js]: searchValue, fromIndex

**Pull request merged**